report/README.md

```diff
@@ -0,0 +1,49 @@
+# Test Suite Runner
+
+This folder contains scripts and docker definitions that can run the suite against an implementation.  The implementations are configured in _implementations.json_.
+
+## Add your implementation
+
+Setting up your application requires several steps.
+
+### Create a CLI
+
+To add your implementation, you'll need to start by creating a CLI application that takes the following parameters:
+
+- `--schema <PATH#POINTER>` - The relative URI to the schema, e.g. `../tests/draft2019-09/additionalItems.json#/0/schema`.
+- `--instance <PATH#POINTER>` - The relative URI to the data instance, e.g. `../tests/draft2019-09/additionalItems.json#/0/tests/0/data`.
```

Comment:
Doing it this way would also mean we could specify that CLIs should accept JSON terminated by line delimiters so we don't pay startup cost of each CLI repeatedly.

Reply:
> Doing it this way would also mean we could specify that CLIs should accept JSON terminated by line delimiters so we don't pay startup cost of each CLI repeatedly.

We'd have to minimize all of the test files to remove line delimiters.  I'm not sure I can do that in a bash script.

Given that I'm learning as I go, all of this is a tall order.

Reply:
Do you mean newlines?
Two NL in a row could be used to signal the end of a json blob, rather than removing all newlines and passing the data as a single line.